results.html

```diff
@@ -2,42 +2,201 @@
+              <iframe
+                src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d239283.1392350707!2d-87.01468175940212!3d20.432348997106995!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x8f4ef8efb0d19327%3A0x342ebcae5c018bfc!2sCozumel!5e0!3m2!1sen!2sus!4v1657560021697!5m2!1sen!2sus"
+                width="100%" height="100%" style="border: 0" allowfullscreen="" loading="lazy"
+                referrerpolicy="no-referrer-when-downgrade"></iframe>
+            </div>
+
+          </div>
+    </div>
+  </div>
+  <div class="rowThree">
+    <div class="row">
+      <div class="col s12 m4 l4">
+        <div class="carousel-info">
+          <div id="Restaurants" class="carousel carousel-slider center">
+            <div class="carousel-item opacity white-text" href="#one!">
+              <h2>Restaurant 1</h2>
+              <p class="white-text">Info 1</p>
+              <img src="/assets/images/restaurant/3-course.jpeg" alt="" class="class">
+            </div>
+            <div class="carousel-item opacity white-text" href="#two!">
+              <h2>Commodities 1</h2>
+              <p class="white-text">Info 2</p>
+            </div>
+            <div class="carousel-item opacity white-text" href="#three!">
+              <h2>Commodities 2</h2>
+              <p class="white-text">Info 3</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col s12 m4 l4">
+        <div class="carousel-info">
+          <div id="housingSalary" class="carousel carousel-slider center">
+            <div class="carousel-item opacity white-text" href="#one!">
+              <h2>Rent an Apartment</h2>
+              <p class="white-text">Info 1</p>
+              <img src="../assets/images/restaurant/inexpensive.avif" alt="" class="class">
+            </div>
+            <div class="carousel-item opacity white-text" href="#two!">
+              <h2>Buying an Apartment/House</h2>
+              <p class="white-text">Info 2</p>
+            </div>
+            <div class="carousel-item opacity white-text" href="#three!">
+              <h2>Average Salary</h2>
+              <p class="white-text">Info 3</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col s12 m4 l4">
+        <div class="carousel-info">
+          <div id="Miscellaneous" class="carousel carousel-slider center">
+            <div class="carousel-item opacity white-text" href="#one!">
+              <h2>Utilities</h2>
+              <p class="white-text">Info 1</p>
+            </div>
+            <div class="carousel-item opacity white-text" href="#two!">
+              <h2>Transportation</h2>
+              <p class="white-text">Info 2</p>
+            </div>
+            <div class="carousel-item opacity white-text" href="#three!">
+              <h2>Childcare</h2>
+              <p class="white-text">Info 3</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
It's unfortunate but so much of this HTML is going to be rendered dynamically, and it doesn't match the classes that I used at all.  None of the APIs work because all they rely on the classes.  I used the classes and the IDs to select the elements in the DOM and now the structure is completely different.  You can see in the JS which classes and IDs I rely on.  All the functions that are named `renderXxxYyy` are traversing the DOM for elements with specific classes and IDs, which now they can't find the elements they need to fill up.